pkg/scheduler/objects/preemption_queue_test.go

```diff
@@ -24,6 +24,35 @@ import (
 	"github.com/apache/yunikorn-core/pkg/common/resources"
 )
 
+var smallestRes = resources.NewResourceFromMap(map[string]resources.Quantity{"first": 5})
+var childRes = resources.NewResourceFromMap(map[string]resources.Quantity{"second": 5})
+var smallestResPlusChildRes = resources.Add(smallestRes, childRes)
+var smallestResDouble = resources.Multiply(smallestRes, 2)
+var smallestResDoublePlusChildRes = resources.Add(resources.Multiply(smallestRes, 2), childRes)
+var smallestResMultiplyByZero = resources.NewResourceFromMap(map[string]resources.Quantity{"first": 0})
+var smallestResMultiplyByMinusOne = resources.Multiply(smallestRes, -1)
+var smallestResMultiplyByMinusOnePlusChildRes = resources.Add(resources.Multiply(smallestRes, -1), childRes)
+var childResMultiplyByZero = resources.NewResourceFromMap(map[string]resources.Quantity{"second": 0})
+var childResMultiplyByMinusOne = resources.Multiply(childRes, -1)
+var childResDouble = resources.Multiply(childRes, 2)
+var smallestResDoublePlusChildResDouble = resources.Add(resources.Multiply(smallestRes, 2), childResDouble)
+var smallestResPlusChildResDouble = resources.Add(smallestRes, childResDouble)
+var smallestResMultiplyByZeroPluschildResMultiplyByZero = resources.Add(smallestResMultiplyByZero, childResMultiplyByZero)
+var smallestResMultiplyByZeroPluschildResMultiplyByMinusOne = resources.Add(smallestResMultiplyByZero, childResMultiplyByMinusOne)
```

Comment:
Wouldn't it be better to just create the resource map in-place wherever it's needed? These variables barely reveal what they contain, the names are way too long sometimes.

Reply:
Agree with this, please simplify.

Reply:
Simplified the variable names like usual test variables. Couple of jiras [YUNIKORN-2848 ](https://issues.apache.org/jira/browse/YUNIKORN-2848 )& [YUNIKORN-2824](https://issues.apache.org/jira/browse/YUNIKORN-2824 )  has been already raised which would re-use these variables eventually.